### ISSUE TITLE
Use TCP for communication

### ISSUE THREAD

**Issue by joboet** [supercoder186/XSharedCockpit#4]
At the moment, the plugin uses UDP for communications, which is problematic, since UDP can drop packets. For datarefs this is mostly unproblematic, but if a packet with a command is lost, the master/slave can go out of sync.
**Fix:** Use TCP, which does not loose packets.

**Comment by supercoder186** [comment]
You are right here, in that TCP is more "safe" to use. I will definitely have to consider moving to TCP soon
